linters/prettier/plugin.yaml

```diff
@@ -28,6 +19,27 @@ lint:
           batch: true
           in_place: true
           formatter: true
+          files:
+            - typescript
+            - yaml
+            - css
+            - sass
+            - html
+            - markdown
+            - json
+            - javascript
+            - graphql
+            - prettier_supported_configs
```

Comment:
Strictly speaking, this is correct. But just wanted to call out that we may see a couple reports when people upgrade because I know a handful of people overrode prettier filetypes in order to support some of the prettier plugins